.travis.yml

```diff
@@ -41,11 +41,6 @@ script:
   - pip install tox
   - ldd /usr/local/lib/x86_64-linux-gnu/libdeltachat.so
   - tox
-  - pip install -e .
-  - pip install sphinx breathe
-  - cd doc
-  - make html
-  - cd ../..
```

Comment:
you will still need to `cd ..`.
Otherwise the script ends in a subdir and the deploy step tries to deploy from there.
I know it's stupid.
Maybe `cd $TRAVIS_BUILD_DIR` at the top of teh deploy step is the better solution.

Reply:
found and applied a more robust solution.